server/data/healthcheck.js

```diff
@@ -15,50 +16,34 @@ function dbCheck() {
   return db.query('SELECT 1 AS ok')
 }
 
-function nomisApiCheck() {
-  return new Promise((resolve, reject) => {
-    superagent
-      .get(`${getHealthcheckUrl()}/health`)
-      .timeout({
-        response: 4000,
-        deadline: 4500,
-      })
-      .end((error, result) => {
-        try {
-          if (error) {
-            logger.error(error.stack, 'Error calling Nomis API')
-            return reject(`${error.status} | ${error.code} | ${error.errno}`)
-          }
+function pdfApiCheck() {
+  return serviceApiCheck('pdf', `${config.pdf.pdfServiceHost}/health`)
+}
 
-          if (result.status === 200) {
-            return resolve('OK')
-          }
+function nomisApiCheck() {
+  return serviceApiCheck('nomis', `${getHealthcheckUrl('nomis')}/health`)
+}
```

Comment:
Is getHealthCheckUrl necessary? It looks as if it is only called with argument 'nomis', so perhaps this could be simplified. `${config.nomis.apiUrl.replace('api','')}/health` perhaps? but maybe that's too compressed.